frontend/common/src/components/UserProfile/ProfileSections/WorkPreferencesSection.tsx

```diff
@@ -86,53 +90,55 @@ const WorkPreferencesSection: React.FunctionComponent<{
           </div>
         )}
 
-        {wouldAcceptTemporary && (
-          <div data-h2-flex-item="base(1of1)">
-            <p>
-              {intl.formatMessage({
-                defaultMessage:
-                  "I would consider accepting a job that lasts for:",
-                id: "Vc9vE7",
-                description:
-                  "Label for what length of position user prefers, followed by colon",
-              })}
-            </p>
-            <ul data-h2-padding="base(0, 0, 0, x1)">
-              <li>
+        {positionDuration &&
+          positionDuration.includes(PositionDuration.Temporary) && (
+            <div data-h2-flex-item="base(1of1)">
+              <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "any duration. (short term, long term, or indeterminate duration)",
-                  id: "uHx3G7",
+                    "I would consider accepting a job that lasts for:",
+                  id: "Vc9vE7",
                   description:
-                    "Label displayed on Work Preferences form for any duration option",
+                    "Label for what length of position user prefers, followed by colon",
                 })}
-              </li>
-            </ul>
-          </div>
-        )}
+              </p>
+              <ul data-h2-padding="base(0, 0, 0, x1)">
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "any duration. (short term, long term, or indeterminate duration)",
```

Comment:
We'll need new copy here, right?  And you'd want to combine the two blocks into a single `ul`?

Reply:
I am hesitant to change copy, I think this change at the moment is more invisible and for ease-of-developer-understanding purposes than changing actual frontend

Reply:
I agree, lets leave this as a backend refactor for now. We have another ticket about improving the copy/inputs, which we should flag again for designers: https://github.com/GCTC-NTGC/gc-digital-talent/issues/4671